quodlibet/quodlibet/config.py

```diff
@@ -203,6 +203,7 @@ def _config_text(text):
 getlist = _config.getlist
 setlist = _config.setlist
 set = _config.set
+unset = _config.unset
```

Comment:
Well, `reset()` resets to the value in `config.py` whereas `unset()` remove the entry in the configuration file.

Anyway, I won't need it anymore if I don't store the values but just the menu selection...